plugins/com.google.cloud.tools.eclipse.ui.test/src/com/google/cloud/tools/eclipse/ui/util/images/SharedImagesTest.java

```diff
@@ -18,17 +18,23 @@
 
 import static org.junit.Assert.assertNotNull;
 
-import com.google.cloud.tools.eclipse.test.util.ui.ShellTestResource;
-import org.junit.Rule;
+import org.eclipse.swt.graphics.Image;
+import org.eclipse.swt.widgets.Display;
 import org.junit.Test;
 
 public class SharedImagesTest {
 
-  @Rule public ShellTestResource shell = new ShellTestResource();
-
   @Test
   public void testCreateRefreshIcon() {
-    assertNotNull(SharedImages.REFRESH_IMAGE_DESCRIPTOR.createImage(shell.getDisplay()));
+    Image image = null;
+    try {
+      image = SharedImages.REFRESH_IMAGE_DESCRIPTOR.createImage(Display.getCurrent());
```

Comment:
I'm guessing this test is trying to verify that the image is resolved, which it doesn't actually do since `createImage()` will return a _missing_ image by default if it cannot be loaded (a red square).  Calling `getImageData()` and checking for null should do the trick.